### PR TITLE
Changes to TestPlanUpdaterModal in the absence of TestPlanTarget

### PR DESCRIPTION
**Pull request by evmiguel**
This PR handles the removal of TestPlanTarget from the TestPlanUpdaterModal. Commits from #419 are included in the PR. The files changed are:

- client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx
- client/components/TestPlanUpdater/queries.js
- client/tests/__mocks__/GraphQLMocks.js

### CODE REVIEW

File: client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx
Comment (alflennik) [inline client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx:380]: Using some recent JS syntax this could be 
```js
if (!runsWithResults?.length) {
```

File: client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx
Comment (alflennik) [inline client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx:376]: This if statement is not longer needed and can be removed!

File: client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx
Comment (alflennik) [inline client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx:383]: I'd actually vote to `return ''`  instead of "There are no test results associated with this report." because when I was using the modal the message confused me. It sounded almost like an error message.

**Review by alflennik — COMMENTED** [review]
Nicely done, I confirmed that this is working with the new schema changes, and the feature of allowing for updating when no test results have been collected yet works as well. Also this is the first time I used the new UI and it looks great. There is just some small comments that should be easy to address.

**Review by alflennik — APPROVED** [review]
Great, looks good! Since this is based on PR 419 let's wait for that to be merged before merging this one.

**Comment by howard-e** [comment]
Thanks again @evmiguel. Closing as changes here were included in #419.

**Pull request closed without merging**
